doc/development/deprecations.rst

```diff
@@ -0,0 +1,83 @@
+.. _deprecations:
+
+Deprecations
+============
+
+Pending Deprecations
+--------------------
+
+* In-place inversion is now deprecated. This includes ``op.inv()`` and ``op.inverse=value``. Please
+  use ``qml.adjoint`` or ``qml.pow`` instead. 
+
+  - Still accessible in v0.27 and v0.28
+  - Removed in v0.29
+
+  Don't use:
+
+  >>> v1 = qml.PauliX(0).inv()
+  >>> v2 = qml.PauliX(0)
+  >>> v2.inverse = True
+
+  Instead use:
+
+  >>> qml.adjoint(qml.PauliX(0))
+  Adjoint(PauliX(wires=[0]))
+  >>> qml.pow(qml.PauliX(0), -1)
+  PauliX(wires=[0])**-1
+  >>> qml.pow(qml.PauliX(0), -1, lazy=False)
+  PauliX(wires=[0])
+  >>> qml.PauliX(0) ** -1
+  PauliX(wires=[0])**-1
+
+* ``QueuingContext`` is renamed ``QueuingManager``. 
+
+  - Still accessible via ``QueuingContext`` in v0.27
+  - Only accessible via ``QueuingManager`` in v0.28
```

Comment:
Same here, th4e second bullet should be future tense ('Will only be accessible via ``QueueingManager`` in v0.28')

Reply:
```suggestion
  - Still accessible via ``QueuingContext`` in v0.27
  - Will only accessible via ``QueuingManager`` in v0.28
```